_scripts/evaluate_doc_retrieval.csv

```diff
@@ -1,9 +1,6 @@
 test_id,query,answer,comments,state_filter_expected,county_filter_expected,commodity_filter_expected,doc_category_filter_expected,retrieved_doc1_expected,retrieved_doc1_page_expected,retrieved_doc2_expected,retrieved_doc2_page_expected,retrieved_doc3_expected,retrieved_doc3_page_expected
 1,are apples produced in yakima county washington?,,,Washington,Yakima,Apples,,SPOI/53_077_90_0054_20230831.pdf,1-6,CP/Apple-Crop-Provisions-11-0054.pdf,,,
 2,almonds?,,"Acceptable answer: Any doc related to almonds should be fine, with priority to more generic docs",,,Almonds,,CP/Almond-Crop-Provisions-08-028.pdf,1-4,,,,
-3,Is there a special provisions document for apples in Yakima county Washington?,,,Washington,Yakima,Apples,SP,SPOI/53_077_90_0054_20230831.pdf,1-6,,,,
-4,Is there a document in special provisions document category for apples in Yakima county Washington?,,,Washington,Yakima,Apples,SP,SPOI/53_077_90_0054_20230831.pdf,1-6,,,,
```

Comment:
@dtourillon-cropguard those tools were triggering the SP doc retrieval tool. Since the scope of the RA chain is changing I felt it was okay to delete them.

Reply:
"those tools" => "those examples" I assume?

ok!